src/library/scala/io/Source.scala

```diff
@@ -167,6 +167,10 @@ object Source {
 
   def fromInputStream(is: InputStream)(implicit codec: Codec): BufferedSource =
     createBufferedSource(is, reset = () => fromInputStream(is)(codec), close = () => is.close())(codec)
+
+  def fromClassPath(resource: String, classLdr: ClassLoader = ClassLoader.getSystemClassLoader)(implicit codec: Codec): BufferedSource =
```

Comment:
Scala has named parameters, so the parameter names are part of the api, so I'd suggest `classLoader` or abbreviating as `loader`, but not just drppng vwls, snc ts s hrd t rmbr whch vwls wr drppd xctl. (Martin gets `mkString`; vowels are for mortals.)